src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -5592,7 +5598,90 @@ GUI_TEST_CLASS_DEFINITION(test_6754) {
+                               QStringList() << "annotations.gb"
+                                             << "sequence.gb");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
+    CHECK_SET_ERR(GTUtilsProjectTreeView::checkItem(os, "Unknown features"),
+                  "No 'Unknown features' object!");
+
+    //    Use context menu on annotation in tree view
+    GTUtilsDialog::waitForDialog(os,
+                                 new ProjectTreeItemSelectorDialogFiller(os,
+                                                                         "annotations.gb",
+                                                                         "Unknown features"));
+    GTUtilsDialog::waitForDialog(os, new CreateObjectRelationDialogFiller(os));
+    GTUtilsDialog::waitForDialog(os,
+                                 new PopupChooserByText(os,
+                                                        QStringList()
+                                                            << "Add"
+                                                            << "Objects with annotations..."));
+    //    On question "Found annotations that are out of sequence range, continue?" answer "Yes"
+    GTUtilsDialog::waitForDialog(os, new MessageBoxDialogFiller(os, QMessageBox::Yes));
+
+    GTUtilsSequenceView::openPopupMenuOnSequenceViewArea(os);
+    //    Check {add-> Objects with annotations} action
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    int seqNum = GTUtilsSequenceView::getSeqWidgetsNumber(os);
+    CHECK_SET_ERR(seqNum == 1, QString("Too many seqWidgets count").arg(seqNum));
+
+    ADVSingleSequenceWidget *seqWgt = GTUtilsSequenceView::getSeqWidgetByNumber(os, 0);
+    CHECK_SET_ERR(GTUtilsCv::isCvPresent(os, seqWgt), QString("No CV for single sequence view."));
+    ADVSequenceObjectContext *ctx = seqWgt->getActiveSequenceContext();
+    CHECK_SET_ERR(ctx != nullptr, QString("No context for single sequence view."));
+    AnnotatedDNAView *aview = ctx->getAnnotatedDNAView();
+    CHECK_SET_ERR(aview != nullptr, QString("No annoview for single sequence view. First check"));
+    QList<ADVSplitWidget *> splitters = aview->getSplitWidgets();
+    CHECK_SET_ERR(splitters.size() == 1,
+                  QString("No splitters for single sequence view. First check"));
+    CircularViewSplitter *splitter = static_cast<CircularViewSplitter *>(splitters[0]);
+    CHECK_SET_ERR(splitter != nullptr, QString("CircularViewSplitter is null"));
+
+    QList<QScrollBar *> scrolls = splitter->findChildren<QScrollBar *>("CircularViewSplitter_horScroll");
+    CHECK_SET_ERR(scrolls.size() == 1, QString("Too many scrolls"));
+    QScrollBar *horScroll = scrolls[0];
 
+    horScroll->setValue(horScroll->value() + 13);
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    GTGlobals::sleep(1000);
+    horScroll->setValue(horScroll->value() + 26);
```

Comment:
Never update QT UI model from non-UI thread.

Reply:
done

Reply:
Looks like some problem with my diff viewer. Can't see any changes in diff